crates/cdk/src/wallet/builder.rs

```diff
@@ -50,6 +52,19 @@ impl WalletBuilder {
         Self::default()
     }
 
+    /// Use HTTP for wallet subscriptions to mint events
+    pub fn prefer_http_subscription(mut self) -> Self {
+        self.prefer_http_subscription = true;
+        self
+    }
```

Comment:
Is this a prefer or a require? WS makes sense to be prefer since we fallback to http if mint does not support ws, but here I think its setting wallet to never use ws?